ranger/core/filter_stack.py

```diff
@@ -149,3 +149,8 @@ def __str__(self):
 
     def decompose(self):
         return [self.subfilter]
+
+
+class TagFilter(BaseFilter):
```

Comment:
Why are we touching `filter_stack`?

Reply:
Because the Travis CI check stated that there was only 1 empty line after the `def __str__()` and the check failed. So I added one more line.